docs/UserGuide.md

```diff
@@ -261,7 +256,7 @@ the rooms that have room clean statuses set to "no".
   ![result for 'find alex 99272758 03-68'](images/findAlex9927275803-68.png)
 
 #### Common mistakes:
-* Searching for guests using partial field characteristics e.g. `170` will not match `170.00`.
+* Searching for guests using partial field characteristics, e.g., `170` will not match `170.00`.
```

Comment:
Should this be E.g.?

Reply:
I think no need because not at the start of a sentence